design/one-pager-provider-nop..md

```diff
@@ -47,52 +70,76 @@ type NopResourceParameters struct {
 
 ### `NopResource` Controller
 
-The controller for this nop type will not call any external APIs during its reconcile loop and just perform the behaviour as specified by the user in the spec. 
-
-#### Setup
-Since default is too long for the testing use case, the sync period should be decreased to a small value like `100ms` using `WithPollInterval`in the Setup function. This will allow controller to set various conditions.
-
-#### Connect
-Currently, there is no requirement for having `ProviderConfig` since we don't have any external API client that we might want to connect. 
-We would just create an external client with no service parameter passed. 
-
-#### Observe
-This is where the main logic will reside that would compare the time elapsed with the time interval passed in the spec. The logic will determine if the status conditions match what the spec says they should be. If they do not match (i.e. we need to update the status conditions), it will return `ResourceUpToDate: false`, indicating that Update should be called to perform the update.
-
-#### Update
-Whenever the update function is called it will match the time elapsed from creation with the `timeAfter` fields of spec and status/condition will be changed accordingly.
-
-#### Delete
+The controller for this nop type will not call any external APIs during its
+reconcile loop and just perform the behaviour as specified by the user in the
+spec. 
+
+#### Setup 
+Since default is too long for the testing use case, the sync period
+should be decreased to a small value like `1s` or less using
+`WithPollInterval`in the Setup function. This will allow controller to set
+various conditions.
+
+#### Connect 
```

Comment:
In a follow up PR to this one, it would be great to make an updated revision to this doc that describes the new functionality and what the challenges were of trying to use the managed reconciler :)